app-i18n/translate-shell/Manifest

```diff
@@ -1 +1 @@
-DIST translate-shell-0.9.6.2.tar.gz 69544 BLAKE2B b49a2c6d7101f5063c8c548c2dcd26ed8b5a310d5e723293b8082bb1bdffbb926fb6f0050953825c9fc4567880cfbf1e9956236db3c28b176db851983c743d0d SHA512 a7bd3fba43485a22c2fdd4261c24579b03aeafefa40863c3b8eb1c814a167b9a6f5700086c1cd411063c66c169875b96d2634ed04f45beb6d53b4725eb7fbea9
+DIST translate-shell-0.9.6.4.tar.gz 69788 SHA256 5ac8fd1a6b772750942f92665966ddf2b825594eb969770aabf16d7a83b3e6b9 SHA512 98c4a329794d38a5d22d98bf7bb821f1adc3e42dac96d2fd791e7cf64d785458d0f6006e83b21497f259525b1a70f306ab69090d3a12996610f742323a371147 WHIRLPOOL 53a9561738356d3ff101bd11ef96792d5cba5419a6db525a467fc3ff924ce97174f337ff8b7359ed3b58466280cf712d043973d48310c21c91e9914f934d7f08
```

Comment:
1. I think you want `Closes:`, not `Bug:` here.
2. Please split removal from bump into separate commits.
3. This is the wrong hash set, so please regenerate the Manifest. You will need to remove the old entry to make Portage do the right thing.